cmux-tui/bindings/verify_github_environment_policy.py

```
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+"""Verify the protection policy on a GitHub Actions environment."""
+
+from __future__ import annotations
+
+import argparse
+import http.client
+import json
+import os
+import re
+import sys
+from typing import Any, Mapping, Optional
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)